wfc3tools/wf3cte.py

```diff
@@ -105,12 +36,8 @@ def wf3cte(input, output=None, parallel=True, verbose=False, log_func=print):
         - a single filename (``iaa012wdq_raw.fits``)
         - a Python list of filenames
         - a partial filename with wildcards (``*raw.fits``)
-        - filename of an ASN table (``*asn.fits``)
         - an at-file (``@input``)
 
-    output : str, default=None
-        Name of the output FITS file.
-
     parallel : bool, default=True
         If True, run the code with OpemMP parallel processing turned on for the
```

Comment:
OpenMP

Reply:
Addressed in 6101480.

Reply:
Was this only addressed in `wf3cte.rst`? I'm having trouble finding the edit for `wf3cte.py`

Reply:
See the bottom of the page for the few changes in wf3cte.py.  The output parameter was removed from the function signature and the docstring.  The parameter was never actually used in the Python wrapper.